### ISSUE TITLE
Column not found: 1054 Unknown column 'is_sticky' in 'order clause'

### ISSUE THREAD

**Issue by ghost** [FriendsOfFlarum/frontpage#12]
Disabling the extension fixed the bugs.

```
Next Illuminate\Database\QueryException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'is_sticky' in 'order clause' (SQL: select * from `users` where `id` in (select `id` from `users` where `username` like recapt%) order by `is_sticky` desc, `frontpage` desc, `frontdate` desc, `last_posted_at` desc limit 6) in /var/www/html/flarum/vendor/illuminate/database/Connection.php:669
Stack trace:
#0 /var/www/html/flarum/vendor/illuminate/database/Connection.php(629): Illuminate\Database\Connection->runQueryCallback()
#1 /var/www/html/flarum/vendor/illuminate/database/Connection.php(338): Illuminate\Database\Connection->run()
#2 /var/www/html/flarum/vendor/illuminate/database/Query/Builder.php(2149): Illuminate\Database\Connection->select()
#3 /var/www/html/flarum/vendor/illuminate/database/Query/Builder.php(2137): Illuminate\Database\Query\Builder->runSelect()
#4 /var/www/html/flarum/vendor/illuminate/database/Query/Builder.php(2609): Illuminate\Database\Query\Builder->Illuminate\Database\Query\{closure}()
#5 /var/www/html/flarum/vendor/illuminate/database/Query/Builder.php(2138): Illuminate\Database\Query\Builder->onceWithColumns()
#6 /var/www/html/flarum/vendor/illuminate/database/Eloquent/Builder.php(546): Illuminate\Database\Query\Builder->get()
#7 /var/www/html/flarum/vendor/illuminate/database/Eloquent/Builder.php(530): Illuminate\Database\Eloquent\Builder->getModels()
#8 /var/www/html/flarum/vendor/flarum/core/src/User/Search/UserSearcher.php(75): Illuminate\Database\Eloquent\Builder->get()
#9 /var/www/html/flarum/vendor/flarum/core/src/Api/Controller/ListUsersController.php(81): Flarum\User\Search\UserSearcher->search()
#10 /var/www/html/flarum/vendor/flarum/core/src/Api/Controller/AbstractSerializeController.php(96): Flarum\Api\Controller\ListUsersController->data()
#11 /var/www/html/flarum/vendor/flarum/core/src/Http/RouteHandlerFactory.php(38): Flarum\Api\Controller\AbstractSerializeController->handle()
#12 /var/www/html/flarum/vendor/flarum/core/src/Http/Middleware/DispatchRoute.php(65): Flarum\Http\RouteHandlerFactory->Flarum\Http\{closure}()
#13 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Flarum\Http\Middleware\DispatchRoute->process()
#14 /var/www/html/flarum/vendor/flarum/core/src/Http/Middleware/SetLocale.php(50): Laminas\Stratigility\Next->handle()
#15 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Flarum\Http\Middleware\SetLocale->process()
#16 /var/www/html/flarum/vendor/flarum/core/src/Http/Middleware/CheckCsrfToken.php(37): Laminas\Stratigility\Next->handle()
#17 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Flarum\Http\Middleware\CheckCsrfToken->process()
#18 /var/www/html/flarum/vendor/flarum/core/src/Http/Middleware/AuthenticateWithHeader.php(55): Laminas\Stratigility\Next->handle()
#19 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Flarum\Http\Middleware\AuthenticateWithHeader->process()
#20 /var/www/html/flarum/vendor/flarum/core/src/Http/Middleware/AuthenticateWithSession.php(32): Laminas\Stratigility\Next->handle()
#21 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Flarum\Http\Middleware\AuthenticateWithSession->process()
#22 /var/www/html/flarum/vendor/flarum/core/src/Http/Middleware/RememberFromCookie.php(51): Laminas\Stratigility\Next->handle()
#23 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Flarum\Http\Middleware\RememberFromCookie->process()
#24 /var/www/html/flarum/vendor/flarum/core/src/Http/Middleware/StartSession.php(61): Laminas\Stratigility\Next->handle()
#25 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Flarum\Http\Middleware\StartSession->process()
#26 /var/www/html/flarum/vendor/flarum/core/src/Api/Middleware/FakeHttpMethods.php(29): Laminas\Stratigility\Next->handle()
#27 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Flarum\Api\Middleware\FakeHttpMethods->process()
#28 /var/www/html/flarum/vendor/flarum/core/src/Http/Middleware/ParseJsonBody.php(28): Laminas\Stratigility\Next->handle()
#29 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Flarum\Http\Middleware\ParseJsonBody->process()
#30 /var/www/html/flarum/vendor/flarum/core/src/Http/Middleware/HandleErrors.php(57): Laminas\Stratigility\Next->handle()
#31 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Flarum\Http\Middleware\HandleErrors->process()
#32 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/MiddlewarePipe.php(84): Laminas\Stratigility\Next->handle()
#33 /var/www/html/flarum/vendor/middlewares/request-handler/src/RequestHandler.php(84): Laminas\Stratigility\MiddlewarePipe->process()
#34 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Middlewares\RequestHandler->process()
#35 /var/www/html/flarum/vendor/middlewares/base-path-router/src/BasePathRouter.php(97): Laminas\Stratigility\Next->handle()
#36 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Middlewares\BasePathRouter->process()
#37 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Middleware/OriginalMessages.php(42): Laminas\Stratigility\Next->handle()
#38 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Laminas\Stratigility\Middleware\OriginalMessages->process()
#39 /var/www/html/flarum/vendor/middlewares/base-path/src/BasePath.php(53): Laminas\Stratigility\Next->handle()
#40 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/Next.php(61): Middlewares\BasePath->process()
#41 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/MiddlewarePipe.php(84): Laminas\Stratigility\Next->handle()
#42 /var/www/html/flarum/vendor/laminas/laminas-stratigility/src/MiddlewarePipe.php(73): Laminas\Stratigility\MiddlewarePipe->process()
#43 /var/www/html/flarum/vendor/laminas/laminas-httphandlerrunner/src/RequestHandlerRunner.php(96): Laminas\Stratigility\MiddlewarePipe->handle()
#44 /var/www/html/flarum/vendor/flarum/core/src/Http/Server.php(42): Laminas\HttpHandlerRunner\RequestHandlerRunner->run()
#45 /var/www/html/flarum/index.php(26): Flarum\Http\Server->listen()
#46 {main}  

```

**Comment by karaok491** [comment]
I'm guessing you don't have `flarum/sticky` enabled/installed?

**Comment by ghost** [comment]
active. Could the error be due to the discussion I pinned to the front page?

![image](https://user-images.githubusercontent.com/55854510/98872735-1ae06d80-2488-11eb-91af-79451a97a82f.png)

```
# php flarum info
Flarum core 0.1.0-beta.14.1
PHP version: 7.4.12
Loaded extensions: Core, date, libxml, openssl, pcre, zlib, filter, hash, pcntl, readline, Reflection, SPL, session, standard, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, json, exif, mysqlnd, PDO, Phar, SimpleXML, sockets, sqlite3, tokenizer, xml, xmlwriter, xsl, mysqli, pdo_mysql, pdo_sqlite, xmlreader, zip
+---------------------------------+------------------+--------+
| Flarum Extensions               |                  |        |
+---------------------------------+------------------+--------+
| ID                              | Version          | Commit |
+---------------------------------+------------------+--------+
| flarum-approval                 | v0.1.0-beta.14   |        |
| flarum-bbcode                   | v0.1.0-beta.12   |        |
| flarum-emoji                    | v0.1.0-beta.14   |        |
| flarum-flags                    | v0.1.0-beta.14.1 |        |
| flarum-likes                    | v0.1.0-beta.14   |        |
| flarum-lock                     | v0.1.0-beta.14   |        |
| flarum-markdown                 | v0.1.0-beta.14   |        |
| flarum-mentions                 | v0.1.0-beta.14   |        |
| flarum-sticky                   | v0.1.0-beta.14   |        |
| flarum-subscriptions            | v0.1.0-beta.14   |        |
| flarum-suspend                  | v0.1.0-beta.14   |        |
| flarum-tags                     | v0.1.0-beta.14   |        |
| flarum-lang-english             | v0.1.0-beta.14.1 |        |
| flarum-statistics               | v0.1.0-beta.14   |        |
| fof-user-bio                    | 0.3.1            |        |
| tolgaaaltas-lang-turkish        | 0.14.1           |        |
| v17development-seo              | 1.4              |        |
| fof-follow-tags                 | 0.5.1            |        |
| fof-formatting                  | 0.2.0            |        |
| fof-split                       | 0.5.0            |        |
| tolgaaaltas-turkish             | 0.14.0           |        |
| nearata-signup-confirm-password | v2.1.0           |        |
| fof-forum-statistics-widget     | 0.3.0            |        |
| nearata-copy-code-to-clipboard  | v1.1.0           |        |
| fof-drafts                      | 0.2.0            |        |
| fof-best-answer                 | 0.2.0            |        |
| fof-links                       | 0.4.0            |        |
| fof-linguist                    | 0.4.4            |        |
| fof-terms                       | 0.5.0            |        |
| fof-merge-discussions           | 0.4.0            |        |
| fof-spamblock                   | 0.3.0            |        |
| fof-stopforumspam               | 0.3.1            |        |
| fof-geoip                       | 0.2.0            |        |
| fof-ban-ips                     | 0.3.0            |        |
| matteocontrini-imgur-upload     | v3.4.1           |        |
| fof-reactions                   | 0.4.2            |        |
| fof-profile-image-crop          | 0.2.0            |        |
| fof-pages                       | 0.5.0            |        |
| fof-nightmode                   | 0.6.0            |        |
| fof-github-autolink             | 0.1.4            |        |
| kyrne-shout                     | 0.2.4-beta.1     |        |
| clarkwinkelmann-author-change   | 0.2.1            |        |
| fof-filter                      | 0.1.0-beta.3     |        |
| fof-username-request            | 0.3.0            |        |
| nomiscz-auth-steam              | v0.2.0           |        |
| fof-oauth                       | 0.1.0            |        |
| michaelbelgium-discussion-views | v5.0.0           |        |
| michaelbelgium-profile-views    | v4.0.0           |        |
| fof-subscribed                  | 0.3.0            |        |
| fof-analytics                   | 0.10.0           |        |
| askvortsov-categories           | v1.1.0           |        |
+---------------------------------+------------------+--------+
Base URL: https://dursuncanpoyraz.com.tr/flarum
Installation path: /var/www/html/flarum
Debug mode: off
```

**Comment by karaok491** [comment]
any outstanding migrations? 

Can you confirm if the database `discussions` table actually has a column `is_sticky`?

**Comment by ghost** [comment]
> `is_sticky` tinyint(1) NOT NULL DEFAULT 0,
> KEY `discussions_is_sticky_created_at_index` (`is_sticky`,`created_at`),
>

**Comment by karaok491** [comment]
Addressed in `0.2.1`
